test/tests/val-2f/comparison_val-2f.py

```diff
@@ -0,0 +1,135 @@
+import matplotlib.pyplot as plt
```

Comment:
Please update this script for the new case. The RMSPE will not be great at first since your model is not completed, but it's OK, because we'll see the improvement as you add new physics.